bookkeeper-server/src/test/java/org/apache/bookkeeper/bookie/BookieInitializationTest.java

```diff
@@ -821,6 +882,8 @@ public void testBookieStartException() throws Exception {
         loggerOutput.expect((List<LoggingEvent> logEvents) -> {
```

Comment:
We'd better assert startFuture and startFuture2 separately.

Reply:
have done.